### DISCUSSION TITLE
Nota as todo/project management app

### DISCUSSION THREAD

**Discussion by work-leonid** [notaapp/talk#273] [Questions]
I like the http://taskpaper.com approach and I think nota would work great instead of taskpaper.com. 

I'm missing a few things
- show all open checkboxes
- bind the checkboxes to the date/time

Maybe I can make an extension for that? I'm not sure which way to look at it.

**Comment by astoilkov** [comment]
Few questions:
- "show all open checkboxes" — do you mean having a dialog/popup where you can search all tasks in your workspace?
- "bind the checkboxes to the date/time" what will that do? I'm trying to understand this. What will be the difference between no binding and just putting a date as part of the task text and binding it?

Thanks!

**Reply by work-leonid** [reply]
> do you mean having a dialog/popup where you can search all tasks in your workspace?

Yes. Checkboxes can be in different documents and it would be great to be able to find them all. 

>"bind the checkboxes to the date/time"

The ability to receive notifications or sort tasks by date. But I understand that this already goes beyond the text application

**Reply by astoilkov** [reply]
> Yes. Checkboxes can be in different documents and it would be great to be able to find them all.

Cool. We can make it easier to find tasks.

> The ability to receive notifications or sort tasks by date. But I understand that this already goes beyond the text application

Yes, this is hard. However, it may be something we may consider in the future.

**Reply by webwesen** [reply]
@astoilkov - for inspiration: please take a look at how Dynalist does it with search queries "is:checklist" - extremely powerful

**Comment by work-leonid** [comment]
Maybe someone is already using nota for their tasks? 

I'm now thinking I can just make a document with all the tasks and link to projects/documents via wiki links. But it would be more convenient if there were minimal automation options

**Reply by waynezhang** [reply]
Seconds this. I'm using Logseq for task management. I put tasks in daily note and use some query to collect them in a single page. That's too much for nota maybe but really like to see Nota support some extended syntax like TODO / DOING / DONE and can search them very quickly.
